Marlin/Marlin_main.cpp

```diff
@@ -4982,32 +4988,26 @@ inline void gcode_G28() {
 
   #if ENABLED(DELTA_AUTO_CALIBRATION)
     /**
-     * G33: Delta '4-point' auto calibration iteration
-     *
-     * Usage: G33 <Cn> <Vn>
-     *
-     *  C  (default) = Calibrate endstops, height and delta radius
-     *
-     *  -2, 1-4: n x n probe points, default 3 x 3
-     *
-     *    1: probe center
-     *       set height only - useful when z_offset is changed
-     *    2: probe center and towers
-     *       solve one '4 point' calibration
-     *   -2: probe center and opposite the towers
-     *       solve one '4 point' calibration
-     *    3: probe 3 center points, towers and opposite-towers
-     *       averages between 2 '4 point' calibrations
-     *    4: probe 4 center points, towers, opposite-towers and itermediate points
-     *       averages between 4 '4 point' calibrations
-     *
-     *  V  Verbose level (0-3, default 1)
-     *
-     *    0: Dry-run mode: no calibration
-     *    1: Settings
-     *    2: Setting + probe results
-     *    3: Expert mode: setting + iteration factors (see Configuration_adv.h)
-     *       This prematurely stops the iteration process when factors are found
+     * G33 - Delta '4-7-point' auto calibration (Requires DELTA)
+     * 
+     * Usage:
+     *   G33 <Cn> <Vn>
+     *   
+     *     Cn = n=-7 -> +7 : n*n probe points
+     *          calibrates height ('1 point'), endstops, and delta radius ('4 points') 
+     *          and calibrates tower angles with n >= 3 ('7+ points')
+     *          n=0  <default>
+     *          n=1  probes center / sets height only
+     *          n=-1 same but 1 iteration only
+     *          n=2  probes center and towers / sets height, endstops and delta radius
+     *          n=-2 same but opposite towers
+     *          n=3  probes all points: center, towers and opposite towers / sets all
+     *          n>3  probes all points multiple times and averages
+     *          n<=3 same but tower angle calibration disabled
```

Comment:
It's not even one iteration; just one quick probe and done ;) But why make an exception for 1-point calibration: P1 and P1 A, then P2 and P2 O, P3, P4,... P7 and P3 T, P4 T... P7 T there's more logic to it. And people who do not remember all those parameters just use P1 to P7 the A, O and T parameters are more 'expert level' anyhow.